lib/player.js

```diff
@@ -1456,8 +1456,13 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
           this.config_.manifest.retryParameters,
           has.mimeType);
     }
+    const manifestConfig = this.config_.manifest;
```

Comment:
Oh, wait, I misread this.
This isn't a copy, it's just a reference. So this will still be changing the manifest.
We have a utility for copying objects that you can use, `shaka.util.ObjectUtils.cloneObject`.

Reply:
Fixed. Sorry about that :(